components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { Button } from '../Button/Button';
+import { cn } from '@/utils/utils';
+import { EntryStatus } from '../EntryStatus/EntryStatus';
+import { ILeagueEntriesProps } from './LeagueEntries.interface';
+import * as React from 'react';
+
+const LeagueEntries = React.forwardRef<HTMLDivElement, ILeagueEntriesProps>(
+  ({ entryNumber, isEliminated = false, isPickSet = false }, ref) => (
+    <div
+      data-testid="LeagueEntryContainerCard"
+      className={cn(
+        'LeagueEntryContainerCard grid h-20 min-w-fit grid-cols-2 justify-between rounded-lg border bg-card p-4 text-card-foreground shadow-sm outline dark:border-zinc-800',
+        isEliminated ? 'bg-zinc-100 dark:bg-zinc-900' : 'transparent',
+      )}
+      ref={ref}
+    >
+      <section
+        className="LeagueEntryHeader flex items-center gap-2 md:gap-12"
+        data-testid="LeagueEntryHeader"
+      >
+        <h4
+          data-testid="LeagueEntryNumber"
+          className={cn(
+            'LeagueEntryNumber text-xl font-semibold leading-none tracking-tight',
+            isEliminated ? 'opacity-50' : '',
```

Comment:
Based on the design, isn't the entire card supposed to have an opacity?

Reply:
Hmm not from what I can tell. Below is the snippet from Figma when selecting the entire entry container.

![CleanShot 2024-06-18Arc001225@2x](https://github.com/LetsGetTechnical/gridiron-survivor/assets/40150036/bbe439bd-d1dd-468d-a127-95f221a77be8)

Reply:
The design may not always be accurate to how you would and can code it for simplicity. Please update.